rst_files/introduction_to_types.rst

```diff
@@ -171,12 +175,14 @@ While you will never have a ``Real`` number directly in memory, the abstract typ
 .. But we've also seen references to types such as ``Real`` and ``AbstractFloat``
 
 
+**can we think of a different subheading here, it's the same as above**
```

Comment:
I think the subheading here could just be removed?

Reply:
Sorry, I looked at the source again for this and take back my comment.

@arnavs @natashawatkins  Thoughts on calling this subsection `Exploring the Type Hierarchy`?

Reply:
Since the whole section is about types like `Number` and `Int64`, how about something like `Example: Numerical Types`?